PostGreKeyRotation/Deployment/helm/templates/secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+data:
+  .dockerconfigjson: eyJhdXRo...
```

Comment:
looks like a tructaed value, should probable replace with something like <GET THIS FROM YOUR DOCKER CONFIG> or the like?

Reply:
Let's leave it for now, just not sure what to put here anyways.